tests/test.vader

```diff
@@ -1432,3 +1432,87 @@ Execute (cleanup):
   nmap S <Plug>Sneak_S
   call SneakReset()
 
+###########################################################
+# Continuous characters edge cases
+
+Given:
+  aaa aaa aaa
+  aaaaaaaaaaa
+  aaa aaa aaa
+
+# considering it's on a match at start, the next one is on the next word
+Do (forward continuous characters not divisible by match length):
+  saar_
+Expect:
+  aaa _aa aaa
+  aaaaaaaaaaa
+  aaa aaa aaa
+
+Do (forward long continuous characters not divisible by match length):
+  jsaar_
+Expect:
+  aaa aaa aaa
+  aa_aaaaaaaa
+  aaa aaa aaa
+
+Do (backwards continuous characters not divisible by match length):
+  wwSaar_
+Expect:
+  aaa a_a aaa
+  aaaaaaaaaaa
+  aaa aaa aaa
+
+# cursor is on an existing match *end*, the first backwards motion goes to this match start
+Do (backwards long continuous characters not divisible by match length):
+  j$Saar_
+Expect:
+  aaa aaa aaa
+  aaaaaaaaa_a
+  aaa aaa aaa
```

Comment:
This actually is a little bit strange, especially in operator-pending. But it's how 2-char sneak has worked all along, hence the comment.

Consider:

```
foobar foobar
           ^
```

`dSba`

```
foobar fooar
          ^
```